onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -0,0 +1,224 @@
+// Copyright(C) 2019 Intel Corporation
+// Licensed under the MIT License
+
+#include <map>
+#include <string>
+#include <memory>
+#include <sstream>
+#include <fstream>
+
+#include <inference_engine.hpp>
+// IE defines a macro 'OPTIONAL' that conflicts the remaining headers using MSVC
+#if defined(_MSC_VER)
+#undef OPTIONAL
+#endif
+
+#include <ngraph/frontend/onnx_import/onnx.hpp>
+#include <ngraph/pass/convert_fp32_to_fp16.hpp>
+
+#include "core/session/onnxruntime_cxx_api.h"
+#include "core/graph/graph.h"
+#include "core/common/logging/logging.h"
+
+#include "backend_utils.h"
+
+namespace onnxruntime {
+namespace openvino_ep {
+namespace backend_utils {
+
+#ifndef NDEBUG
+bool IsDebugEnabled() {
+#ifdef __linux__
+  return (std::getenv("UEP_ENABLE_DEBUG") != nullptr);
+#else
+  return false;
```

Comment:
why not support windows debug dump?
see https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/providers/ngraph/ngraph_custom_op.cc#L32 
as example